### DISCUSSION TITLE
Unable to turn off Naomi free play

### DISCUSSION THREAD

**Discussion by albmm74** [flyinghead/flycast#1492] [General]
Hi. I don't get turning off free play on standalone version: I dissable it on options menu, and I remove the .nvmem file of every Naomi title, but after that all Naomi games still starts in free play mode.

This success with the latest .appimage (2.3) on Ubuntu 22.04.4.

What I'm doing wrong? Thanks.

**Comment by flyinghead** [comment]
Remove the .eeprom file instead

**Reply by albmm74** [reply]
Thanks, but there's no eeprom file for games with this issue. I have adviced that it success with tate games only, and success in the RA core too.

Any other possible solution?. Thanks again.

**Comment by flyinghead** [comment]
Then it's probably because Flycast includes an Free Play eeprom for this game. Since you don't say which games you tried, I can't confirm it.
Anyway, to disable free play, go to the test/service menu and change the coin assignments.

**Reply by albmm74** [reply]
Thanks @flyinghead

This success with:

- Ikaruga
- Psyvariar 2
- Trizeal
- Under Defeat
- Virtua Tennis 1

I have change it by game. 

Any way to reset the number of coins inserted? The coins keep between game resets.

Thanks.

**Comment by albmm74** [comment]
Hi.

Any way to reset the coins to 0?

**Comment by flyinghead** [comment]
Restore the .eeprom with 0 coins inserted I suppose

**Comment by albmm74** [comment]
Thanks @flyinghead . Restoring the .eeprom does not work, but deleting the .nvmem file, before launching any game, did the trick. Do you know any way to avoid generating .nvmem files (any included option)?

I'm ussing EmulationStation-DE for emulators launch; I did an script for Naomi/Atomiswave launch, including a line for .nvmem deleting.

But I adviced a fail with v2.3: for any reason, I launch a game, and works; but after that, trying to launch the same game again does not start and freeezes my system. This is solved returning to v2.2.

Thanks again.

**Comment by albmm74** [comment]
@flyinghead : do the RA core reads per game .eeprom files?

**Comment by flyinghead** [comment]
The RA core and standalone are identical in that regard

**Reply by albmm74** [reply]
Thanks. I do not get the standalone .eeproms working on RA; and for any reason, I do not get the service and test buttons working. Tested with the nightly RA and Flycast core from 2024-04-01.
